contrib/CMakeLists.txt

```diff
@@ -60,8 +60,8 @@ if (USE_INTERNAL_ZLIB_LIBRARY)
     target_compile_definitions (zlib PUBLIC ZLIB_COMPAT WITH_GZFILEOP)
     target_compile_definitions (zlibstatic PUBLIC ZLIB_COMPAT WITH_GZFILEOP)
     if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "AMD64")
-       target_compile_definitions (zlib PUBLIC X86_64)
-       target_compile_definitions (zlibstatic PUBLIC X86_64)
+       target_compile_definitions (zlib PUBLIC X86_64 X86_NOCHECK_SSE2 UNALIGNED_OK UNROLL_LESS)
```

Comment:
What is UNROLL_LESS?

Reply:
these flags copied from
https://github.com/ClickHouse-Extras/zlib-ng/blob/9173b89d46799582d20a30578e0aa9788bc7d6e1/CMakeLists.txt#L402
UNROLL_LESS not used in headers, we can remove from here